sdk/python/feast/infra/feature_servers/app.py

```diff
@@ -0,0 +1,13 @@
+from mangum import Mangum
+
+import feast
+from feast.feature_server import get_app
+
+# TODO: replace this with an actual feature repo config deserialized from the env variables
```

Comment:
I'm confused, why aren't we doing this now? Don't we want to build a Docker image that will actually construct the RepoConfig from environment variables?

Reply:
Yes, that's a separate task though. I'll send out another PR for that (just trying to iterate quickly)